java-chassis-dependencies/pom.xml

```diff
@@ -324,7 +324,7 @@
       <dependency>
         <groupId>org.hibernate</groupId>
         <artifactId>hibernate-validator-annotation-processor</artifactId>
-        <version>5.2.2.Final</version>
+        <version>6.0.2.Final</version>
```

Comment:
license file need to be updated too.

Reply:
Please update the version in the https://github.com/apache/incubator-servicecomb-java-chassis/blob/master/java-chassis-distribution/src/release/LICENSE#L349

Reply:
Fixed

Reply:
It looks like the annotation processor is option one,  it's just for the IDE validation.  Do we need to ship it?